charts/template-api/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "template-api.fullname" . }}-test-connection"
+  labels:
+{{ include "template-api.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args:  ['{{ include "template-api.fullname" . }}:{{ .Values.service.port }}']
+  restartPolicy: Never
```

Comment:
Expected behaviour? https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#example-states

You can add values to customize command and args 😇 Or enable ou disable this feature